src/meta/src/rpc/service/cluster_service.rs

```diff
@@ -48,8 +48,8 @@ where
         request: Request<AddWorkerNodeRequest>,
     ) -> Result<Response<AddWorkerNodeResponse>, Status> {
         let req = request.into_inner();
-        let worker_type = req.get_worker_type().map_err(tonic_err)?;
-        let host = req.get_host().map_err(tonic_err)?.clone();
+        let worker_type = req.get_worker_type().map_err(meta_error_to_tonic)?;
+        let host = req.get_host().map_err(meta_error_to_tonic)?.clone();
```

Comment:
I think it's okay to always `unwrap` here.

Reply:
I think we can create a new issue to make a RFC about `unwrap` in source code.